less/overrides/buttons.less

```diff
@@ -150,7 +150,9 @@
     padding: 28px 20px;
     border-radius: @border-radius-large;
   }
-
+  &.btn-round-border {
+    border-radius: 100px;
```

Comment:
this class is used to create semicircle-like rounded corners for buttons: 
<img width="300" alt="screen shot 2015-10-26 at 11 20 12 am" src="https://cloud.githubusercontent.com/assets/3991735/10738224/8d7fc1b4-7bd3-11e5-9459-c0496f45dd5c.png">

i'm hoping there's a way to do this without using an arbitrarily large number. a percentage, unfortunately, does not declare the same styling